app/__init__.py

```diff
@@ -0,0 +1,16 @@
+from flask_restplus import Api
+from flask import Blueprint
+
+from .main.controller.user_controller import api as user_ns
+from .main.controller.auth_controller import api as auth_ns
+
+blueprint = Blueprint('api', __name__)
+
+api = Api(blueprint,
+          title='FLASK RESTPLUS API BOILER-PLATE WITH JWT',
```

Comment:
Please put actual title and other details.

For example: title can be "Flask Restplus Common Backend For Tech-Ambit"